include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -2165,6 +2180,77 @@ namespace VectorTools
 
   namespace
   {
+    // keep first argument is non-reference since we use it
+    // with 1e-8 * number
+    template <typename number1, typename number2>
+    bool real_part_bigger_than(const number1 a,
+                               const number2 &b)
+    {
+      return a > b;
+    }
+
+    template <typename number1, typename number2>
+    bool real_part_bigger_than(const std::complex<number1> a,
+                               const std::complex<number2> &b)
+    {
+      Assert(std::abs(a.imag()) < 1e-15 , ExcInternalError());
+      Assert(std::abs(b.imag()) < 1e-15 , ExcInternalError());
+      return a.real() > b.real();
+    }
+
+    template <typename number>
+    double min_number()
+    {
+      return std::numeric_limits<number>::min();
```

Comment:
Can you document what you need these for? Why not just stick with `std::numeric_limit<number>::min()` in all of the places where you call this function.